### PR TITLE
Add aimec analysis to standard tests

### PR DESCRIPTION
**Pull request by awirb**
* add plots to report 
* add required functions to ana3Aimec for report generation 
* add aimec ini files for benchmarks
* update com1DFACfg.ini in benchmark for com1DFAPy (flags)

### CODE REVIEW

File: avaframe/log2Report/generateCompareReport.py [hunk 1]
```diff
@@ -26,7 +26,21 @@ def copyPlots(avaName, testName, outDir, plotListRep, rel):
...
 
     return plotPaths
+    
 
+def copyAimecPlots(plotFiles, testName, outDir, plotPaths):
+    """ copy the quick plots to report directory """
+
+    for pDict in plotFiles:
+        for key in pDict:
+            print('key', key, 'file', pDict[key])
```
Comment (fso42) [inline avaframe/log2Report/generateCompareReport.py:36]: This is a debug print, right? -> Please remove

File: avaframe/log2Report/generateCompareReport.py [hunk 2]
```diff
@@ -26,7 +26,21 @@ def copyPlots(avaName, testName, outDir, plotListRep, rel):
...
+            name = os.path.basename(pDict[key])
+            plotName =  os.path.join(outDir, '%s_%s.png' % (testName, name))
+            shutil.copy2(pDict[key], plotName)
+            log.info('Copied: %s to %s' % (pDict[key], plotName))
```
Comment (fso42) [inline avaframe/log2Report/generateCompareReport.py:40]: debug level?

File: avaframe/out3Plot/outAIMEC.py [hunk 3]
```diff
@@ -120,41 +119,45 @@ def visuRunoutComp(rasterTransfo, resAnalysis, plim, newRasters, cfgPath, cfgFla
...
-        ax.plot(dataMax[1, :], s, '--b', label='Max Simulation')
-        ax.plot(dataMean[1, :], s, '-b', label='Mean Simulation')
+    for ax, maxVal, meanVal, titleVal, unitVal in zip(axes.flatten(), dataMax, dataMean, title, unit):
+        ax.plot(maxVal[0, :], s, '--k', label='Max Reference')
+        ax.plot(meanVal[0, :], s, '-k', label='Mean Reference')
+        ax.plot(maxVal[1, :], s, '--b', label='Max Simulation')
+        ax.plot(meanVal[1, :], s, '-b', label='Mean Simulation')
 
-        ax.set_title(title + 'distribution along the path')
+        ax.set_title(titleVal + 'distribution along the path')
```
Comment (fso42) [inline avaframe/out3Plot/outAIMEC.py:146]:
```suggestion
        ax.set_title(titleVal + 'distribution along path')
```

File: avaframe/out3Plot/outAIMEC.py [hunk 4]
```diff
@@ -392,15 +393,23 @@ def visuComparison(rasterTransfo, inputs, cfgPath, cfgFlags):
 
     ax2.set_title(
         'Difference between current and reference in the RunOut area' + '\n' + 'Blue = FN, Red = FP')
```
Comment (fso42) [inline avaframe/out3Plot/outAIMEC.py:395]:
```suggestion
        'Difference current -  reference in runout area' + '\n' + 'Blue = FN, Red = FP')
```

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/log2Report/generateCompareReport.py (hunk 1 above)
Comment (awirb) [reply avaframe/log2Report/generateCompareReport.py:36]: done

File: avaframe/log2Report/generateCompareReport.py (hunk 2 above)
Comment (awirb) [reply avaframe/log2Report/generateCompareReport.py:40]: done

**Review by matthiasto — APPROVED** [review]

**Pull request merged**